contracts/Mocks/PermitMintableERC20.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity >=0.6.12 < 0.9.0;
+
+// SPDX-License-Identifier: MIT
+
+import "@openzeppelin/contracts/drafts/ERC20Permit.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract PermitMintableERC20 is ERC20, ERC20Permit {
```

Comment:
Can just add the Permit functionality to the existing `ERC20` mock since its only adding an extra function